stdlib/multiprocessing/context.pyi

```diff
@@ -120,7 +120,7 @@ class BaseContext:
         @overload
         def get_context(self, method: str) -> BaseContext: ...
 
-    def get_start_method(self, allow_none: bool = ...) -> str: ...
+    def get_start_method(self, allow_none: bool = ...) -> str | None: ...
```

Comment:
I don't think this is right, where do you see None?

Reply:
It is documented as potentially returning `None`: https://docs.python.org/3/library/multiprocessing.html#multiprocessing.get_start_method

Reply:
Seems like we'd ideally have an overload based on allow_none?
```suggestion
    @overload
    def get_start_method(self, allow_none: Literal[False] = ...) -> str: ...
    @overload
    def get_start_method(self, allow_none: bool) -> str | None: ...
```